packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -997,6 +997,17 @@ export const nemo = (model: ModelData): string[] => {
 	return command ?? [`# tag did not correspond to a valid NeMo domain.`];
 };
 
+export const pxia = (model: ModelData): string[] => [
+	`# Installation
+
+pip install pxia
+`,
+	`# Load the model
+
+from pxia import AutoModel
+model = AutoModel.from_pretrained("${model.id}")`,
+];
+
```

Comment:
```suggestion
export const pxia = (model: ModelData): string[] => [
	`from pxia import AutoModel

model = AutoModel.from_pretrained("${model.id}")`,
];

```

we usually don't put the install command in "use my model" snippets